src/Components/EligibilityCard/styles.ts

```diff
@@ -16,12 +16,12 @@ export default StyleSheet.create({
     alignContent: 'center',
   },
   button: {
-    paddingHorizontal: 20,
+    paddingHorizontal: 30,
     alignSelf: 'center',
     height: '90%',
     width: '100%',
     border: 'solid',
-    borderColor: colors.darkGrey,
+    borderColor: colors.midGrey,
     borderWidth: 0.5,
     borderRadius: 5,
     background: colors.white,
```

Comment:
Let's remove the `margin: 5` from here. This was causing the issue of different widths between the Case Summary and Eligibility buttons on the `Case` screen